packages/web/app/.storybook/preview.js

```diff
@@ -1,7 +1,16 @@
 import '../src/index.css';
 
-if (!document.body.className.includes('dark')) {
-  document.body.className += ' ' + 'dark';
+//  Exposes environment variables to the frontend as a JavaScript object.
+window.__ENV ||= {
+  ENVIRONMENT: 'development',
+  APP_BASE_URL: 'http://localhost:3000',
+  GRAPHQL_PUBLIC_ENDPOINT: 'http://localhost:3001/graphql',
+  GRAPHQL_PUBLIC_SUBSCRIPTION_ENDPOINT: 'http://localhost:3001/graphql',
+  GRAPHQL_PUBLIC_ORIGIN: 'http://localhost:3001',
+};
```

Comment:
Idk what a proper way to inject env variables (set `window.__ENV`) for the storybook, any suggestions?